refinery/core/urls.py

```diff
@@ -11,66 +11,53 @@
 from .views import (AnalysesViewSet, DataSetsViewSet, NodeViewSet, OpenIDToken,
                     WorkflowViewSet)
 
+UUID_RE = r'[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}'
+
 urlpatterns = patterns(
     'core.views',
     url(r'^$', 'home', name="home"),
     url(r'^about/$', 'about', name="about"),
     url(r'^contact/$', 'contact', name="contact"),
     url(r'^statistics/$', 'statistics', name="statistics"),
     url(r'^collaboration/$', 'collaboration', name='collaboration'),
-    url(r'^group_invite/(?P<token>'
-        r'[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12})/$',
+    url(r'^group_invite/(?P<token>' + UUID_RE + r')/$',
         'group_invite', name='group_invite'),
     url(r'^users/(?P<query>[\@\.\-\+a-z0-9]+)/$', 'user'),
     # "name" is required for use with the url tag in templates
-    url(r'^users/(?P<query>'
-        r'[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12})/$',
+    url(r'^users/(?P<query>' + UUID_RE + r')/$',
         'user', name="user"),
-    url(r'^users/(?P<uuid>'
```

Comment:
I would rather keep it as is to make it easier for anyone who has read Django docs understand what's going on.